ui/lib/kmip/addon/templates/credentials/show.hbs

```diff
@@ -43,21 +43,17 @@
   <InfoTableRow @label="Serial number" @value={{this.model.id}}>
     <MaskedInput @value={{this.model.id}} @displayOnly={{true}} @allowCopy={{true}} />
   </InfoTableRow>
-  <InfoTableRow @label="Private key" @value={{this.model.privateKey}}>
-    <div class="is-block">
-      <Hds::Alert data-test-warning @type="inline" @color="warning" @class="has-bottom-margin-s" as |A|>
-        <A.Title>Warning</A.Title>
-        <A.Description>You will not be able to access the private key later, so please copy the information below.</A.Description>
-      </Hds::Alert>
-      <MaskedInput
-        @value={{this.model.privateKey}}
-        @name="Private key"
-        @allowCopy={{true}}
-        @displayOnly={{true}}
-        class="is-block"
-      />
-    </div>
-  </InfoTableRow>
+  {{#if this.model.privateKey}}
```

Comment:
🧹 added this conditional as small cleanup - this row was rendering when there was no private key data
<img width="802" alt="Screenshot 2023-09-12 at 9 39 14 AM" src="https://github.com/hashicorp/vault/assets/68122737/2992c7e3-fd46-4080-9e53-091c7852298d">